Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/category/PodcastCategory.kt

```diff
@@ -144,10 +149,11 @@ private fun EpisodeList(episodes: List<EpisodeToPodcast>) {
 fun EpisodeListItem(
     episode: Episode,
     podcast: Podcast,
+    onClick: (String) -> Unit,
     modifier: Modifier = Modifier
 ) {
     ConstraintLayout(
-        modifier = Modifier.clickable { /* TODO */ } then modifier
+        modifier = Modifier.clickable { onClick(episode.uri) } then modifier
```

Comment:
Nit: Is this an intentional violation of concatenating something before the parent modifier?

In the context of https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#elements-accept-and-respect-a-modifier-parameter

Reply:
Fixed, thanks!